src/OneDrive.js

```diff
@@ -310,9 +310,19 @@ class OneDrive extends EventEmitter {
 
   /**
    */
-  async getDriveItemFromShareLink(sharingUrl) {
+  async getDriveItemFromShareLink(sharingUrl, fetchInfo = false) {
     let driveItem = OneDrive.driveItemFromURL(sharingUrl);
     if (driveItem) {
+      if (fetchInfo) {
+        try {
```

Comment:
why not use: `getDriveItem()` 
https://github.com/adobe/helix-onedrive-support/blob/ce86668d97d6795c250438281a47f8d46e448872/src/OneDrive.js#L429

Reply:
Ah, missed that one 🙄 I think I was confused by the parameter name `folderItem`, so I didn't realise this works for other items as well, thanks!